processor/elasticinframetricsprocessor/config_test.go

```diff
@@ -19,13 +19,18 @@ func TestLoadConfig(t *testing.T) {
 	}{
 		{
 			configPath: "config.yaml",
-			id:         component.NewIDWithName(metadata.Type, "default"),
-			expected:   &Config{AddSystemMetrics: true},
+			id:         component.NewID(metadata.Type),
+			expected:   &Config{AddSystemMetrics: true, AddK8sMetrics: true},
+		},
+		{
+			configPath: "config.yaml",
+			id:         component.NewIDWithName(metadata.Type, "2"),
+			expected:   &Config{AddSystemMetrics: true, AddK8sMetrics: true},
 		},
 		{
 			configPath: "config.yaml",
 			id:         component.NewIDWithName(metadata.Type, "with_system_metrics"),
-			expected:   &Config{AddSystemMetrics: true},
```

Comment:
Should we create a new test with_k8s_metrics?

Reply:
Agree

Reply:
done - https://github.com/elastic/opentelemetry-collector-components/pull/14/commits/cf215e8e1fe25354459ee6e668f97a1e9f170a94